rats-tls/cmake/SGXSSL.cmake

```diff
@@ -44,8 +40,15 @@ set(_make_script "${CMAKE_CURRENT_SOURCE_DIR}/intel_sgxssl_make.sh")
 if(SGX)
     file(WRITE "${_make_script}"
 "#!/bin/sh
-cd ${INTEL_SGXSSL_SRC}/Linux
-make
+if [ ! -d ${INTEL_SGXSSL_LIB_PATH} ]; then
+        cd ${INTEL_SGXSSL_SRC}/Linux;
+        make -j$(nproc)
+fi;
+cd ${INTEL_SGXSSL_LIB_PATH}
+if [ ! -e  libsgx_usgxssl.a ]; then
+        cd ${INTEL_SGXSSL_SRC}/Linux;
+        make -j$(nproc)
+fi;
```

Comment:
Plz combine the above conditions into a single one.

Reply:
if [ || ] doesn't work

Reply:
Use '-o'. `if [ ! -d ${INTEL_SGXSSL_LIB_PATH} -o ! -e ${INTEL_SGXSSL_LIB_PATH}/libsgx_usgxssl.a ]; then`